src/main/java/seedu/address/logic/commands/ListProcCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.Client;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.*;
```

Comment:
The code has been of high quality thus far! Just a minor tweak, remember to replace wildcard imports with specific imports instead :).